src/xcache-consistency-check

```diff
@@ -251,8 +251,8 @@ def get_list_of_baskets_in_branch(branch, byte_ranges, is_full_file):
 
 def recursive_branch(branch, byte_ranges, is_full_file):
     list_baskets = []
-    if len(branch.keys()) > 0:
-        for sub_branch in branch.itervalues():
+    if len(list(branch.keys())) > 0:
```

Comment:
```suggestion
    if branch:
```
(if branch is a dictionary).

Reply:
> (if branch is a dictionary)

that was my question too.  It seemed like `branch` was getting used as a dictionary and in that case it's way better to test that directly vs making a list of keys and taking the length.  But i got bored trying to track down exactly what `branch` is.

It apparently is tied to whatever `uproot.open` returns - which is some kind of python Mapping.  I don't know if it's guaranteed to behave like a dict when evaluated like a bool.  If you have a reason to believe it will work, i'm all for it, otherwise i don't want to be responsible for breaking it :)

Reply:
I don't have a reason to believe it will work, though I would consider it a bug in `uproot` if it didn't :-P

Reply:
Ok, well then, I'm committing your suggestion.

And hoping for the best.